nutstash-wallet/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: nutstash-wallet_web_1
+      APP_PORT: 4173
+      PROXY_AUTH_ADD: "false"
+      
+  web:
+    image: ghcr.io/gandlafbtc/nutstash-wallet:0.1.6@sha256:1fc859c57c15918c19774c95325c67e12a10191dc3e2f09c184241dffe478776
+    restart: on-failure
+    stop_grace_period: 1m
+    init: true
```

Comment:
Are these two lines needed? 

`stop_grace_period` is used to set the time that a container tries to gracefully shut down and complete any ongoing tasks. The default value is 10s if `stop_grace_period` is not included. Once this time has expired, the container will be forcefully terminated. Since nutstash is a browser wallet I don't think we need to wait for anything to happen before shutting down, correct?

`init: true` is used to run an initialization process in the container before starting the main process. It doesn't look like we have any entrypoint scripts or anything to run, so we can likely remove this.

```suggestion
```